### PR TITLE
feat: cshl genome informatics 2023 (#2644)

### LINKED ISSUE #2644: CSHL Genome Informatics 2023 event
Thank you for submitting an event to anvilproject.org!

To expedite your event's addition to https://anvilproject.org/events,
please provide the following package information.  You can easily edit this information later by clicking "Improve this page" at the bottom of your event's detail page.

See https://anvilproject.org/guides/content/events-guide for a description of the fields listed below:

## Basics
- Event Title: CSHL Genome Informatics 2023
- Conference short name: Genome Informatics 2023
- Short Description: [max 200 char]
- Event Type: Conference
- Date: December 6, 2023 to December 9, 2023
- Time: all day
- Timezone: ET
- Location: Cold Spring Harbor, NY, USA

### Event Text
The AnVIL team will attend the CSHL Genome Informatics meeting to share about the AnVIL platform with the genomics community. This meeting will draw together researchers focused on genomic data science research, for which AnVIL can be a useful platform to leverage and can offer solutions to some common challenges in the field.

### Background
The next Cold Spring Harbor Laboratory Conference on Genome Informatics will be held at Cold Spring Harbor, New York (this meeting has alternated with the Hinxton meeting started under the joint auspices of CSHL and the Wellcome Trust). The meeting will begin at 7:30 p.m. on Wednesday, December 6, and finish with lunch on Saturday, December 9, 2023.
Tentative Meeting Topics:

* PanGenome
* Genome Assembly and Sequencing Algorithms
* Variant Discovery
* Microbial and Metagenomics
* Single Cell and Spatial Omics
* Functional Genomics

## Event Details

- Conference website: https://meetings.cshl.edu/meetings.aspx?meet=INFO
- Agenda is to be announced.
- Abstract submission is due on September 22, 2023.
- How to register: https://meetings.cshl.edu/meetingsregistration.aspx?meet=INFO&year=23
- Costs:  https://meetings.cshl.edu/meetings.aspx?meet=INFO
- Scholarships supported by the JXTX foundation are available for application through September 15, 2023: https://jxtxfoundation.org/news/2023-04-07-gi/.
- Contact [Natalie Kucher](mailto:nkucher3@jhu.edu) with any questions.
- Hashtag - #gi2023

### PR DESCRIPTION
**Pull request by esthermcdade**
Closes #2644

### CODE REVIEW

File: content/events/cshl-genome-informatics-2023.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+author: "AnVIL"
+conference: "Genome Informatics 2023"
+description: "TThe AnVIL team will attend the CSHL Genome Informatics meeting to share about the AnVIL platform with the genomics community. This meeting will draw together researchers focused on genomic data science research, for which AnVIL can be a useful platform to leverage and can offer solutions to some common challenges in the field."
```
Comment (frano-m) [inline content/events/cshl-genome-informatics-2023.md:4]:
```suggestion
description: "The AnVIL team will attend the CSHL Genome Informatics meeting to share about the AnVIL platform with the genomics community. This meeting will draw together researchers focused on genomic data science research, for which AnVIL can be a useful platform to leverage and can offer solutions to some common challenges in the field."
```

File: content/events/cshl-genome-informatics-2023.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+<event-hero></event-hero>
+
+### Description
+
+The AnVIL team will attend the CSHL Genome Informatics meeting to share about the AnVIL platform with the genomics community. This meeting will draw together researchers focused on genomic data science research, for which AnVIL can be a useful platform to leverage and can offer solutions to some common challenges in the field.
+
+#### Background
+
+The next Cold Spring Harbor Laboratory Conference on Genome Informatics will be held at Cold Spring Harbor, New York (this meeting has alternated with the Hinxton meeting started under the joint auspices of CSHL and the Wellcome Trust). The meeting will begin at 7:30 p.m. on Wednesday, December 6, and finish with lunch on Saturday, December 9, 2023.
+Tentative Meeting Topics:
```
Comment (frano-m) [inline content/events/cshl-genome-informatics-2023.md:23]: @esthermcdade Insert a new line between lines 22 and 23 and view the change on localhost to see the difference.

**Review by frano-m — CHANGES_REQUESTED** [review]
@esthermcdade nice work - there is only one tiny change to be made. Thank you.

**Comment by NoopDog** [comment]
@esthermcdade you will need to rebase this branch.

**Review by frano-m — APPROVED** [review]
Thank you @esthermcdade ⭐

**Pull request merged**